Objects/moduleobject.c

```diff
@@ -866,6 +866,42 @@ _Py_module_getattro(PyModuleObject *m, PyObject *name)
     return _Py_module_getattro_impl(m, name, 0);
 }
 
+static int
+module_setattro(PyModuleObject *mod, PyObject *name, PyObject *value)
+{
+    PyObject *res;
+    int ret;
+    assert(mod->md_dict != NULL);
+    if (value == NULL) {
+        PyObject *delattr;
+        ret = PyDict_GetItemRef(mod->md_dict, &_Py_ID(__delattr__), &delattr);
+        if (ret == -1) {
+            return -1;
+        }
+        if (ret) {
```

Comment:
You can avoid the ret variable:

```suggestion
        PyObject *delattr;
        if (PyDict_GetItemRef(mod->md_dict, &_Py_ID(__delattr__), &delattr) < 0) {
            return -1;
        }
        if (delattr) {
```